### DISCUSSION TITLE
is raspbbery pi 4 and 20 Mbps bandwidth enough for a code-server?

### DISCUSSION THREAD

**Discussion by Slodziak190q** [coder/code-server#3998] [Q&A] [answered]
is raspbbery pi 4 and 20 Mbps bandwidth enough for a code-server?

**Comment by jsjoeio** [comment, marked as answer]
It _should_ be. I have a Pi4 and was able to use code-server on it. I haven't tried 20mbps bandwidth. But you could try and let us know?

**Reply by Slodziak190q** [reply]
J will test it Tomorrow

**Reply by bpmct** [reply]
How'd it go? I've used code-server + RPI4 + limited connection for RPI and it's worked great :)

**Reply by Slodziak190q** [reply]
It works great now j can code everywhere
